src/constants/L2Chain.ts

```diff
@@ -0,0 +1,10 @@
+export enum L2Chain {
```

Comment:
hey @npty let's avoid these enums, especially since the constants can be different from testnet and mainnet

and we should see if there's a way to fetch them programmatically. one way to do that is to see if the response to `getFees` has properties under `destination_native_token`, like this:

`		"destination_native_token": {
			"contract_address": "0x0000000000000000000000000000000000000000",
			"name": "Ethereum",
			"symbol": "ETH",
			"decimals": 18,
			"token_price": {
				"usd": 100
			},
			"gas_price": "0.000000001500000254",
			"gas_price_gwei": "1.500000254",
			"gas_price_in_units": {
				"value": "1500000254",
				"decimals": 18
			},
			"l1_gas_price": "0.000000000014037022",
			"l1_gas_price_gwei": "0.014037022",
			"l1_gas_price_in_units": {
				"value": "14037022",
				"decimals": 18
			},
			"l1_fee_scalar": 1
		},`